.travis.sh

```diff
@@ -86,6 +86,8 @@ case "$JDK" in
   ;;
 9-ea | 9-ea-stable)
   # see https://bugs.openjdk.java.net/browse/JDK-8131041 about "java.locale.providers"
```

Comment:
@marchof I'm not sure that `-DargLine` can be passed via `MAVEN_OPTS` - usually last one is used to pass options that control Maven JVM, while `-D` is argument to Maven invocation